python/sklearn-digits/app.py

```diff
@@ -97,8 +97,9 @@ def evaluate(self, file_path: str) -> Tuple[EvaluateResult, List[EvaluateResultD
                 label_predict.append(result.label)
 
             accuracy = accuracy_score(label_gold, label_predict)
-            p_r_f = precision_recall_fscore_support(label_gold, label_predict)
-            res = EvaluateResult(num, accuracy, p_r_f[0].tolist(), p_r_f[1].tolist(), p_r_f[2].tolist(), {})
+            uniq_labels = sorted(list(set(label_gold)))
```

Comment:
Does it work? I think the label order must not be sorted since we need to correspond them to the output.

Reply:
That's right. I removed `sorted` , so please check it again.